cmd/argo/commands/archivelabel/list.go

```diff
@@ -0,0 +1,29 @@
+package archivelabel
+
+import (
+	"fmt"
+
+	"github.com/argoproj/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/argoproj/argo-workflows/v3/cmd/argo/commands/client"
+	workflowarchivepkg "github.com/argoproj/argo-workflows/v3/pkg/apiclient/workflowarchive"
+)
+
+func NewListCommand() *cobra.Command {
```

Comment:
I think this should be `argo archive labels list` or `argo archive list-labels`.

Reply:
change to

```
argo archive list-labelkeys
argo archive list-labels key_id
```